.evergreen/orchestration/configs/servers/windows/rsa-basic-tls-ocsp-mustStaple.json

```diff
@@ -0,0 +1 @@
+../rsa-basic-tls-ocsp-mustStaple.json
```

Comment:
Symlink, since the files are currently identical

Reply:
Does this work? I remember symlinks causing problems in windows a few years ago.

Reply:
Hm a good question! I'd forgotten this runs in Cygwin on the Windows hosts. Symlinks are supposed to work on Windows with Git on Windows 10 in dev mode, but for simplicity's sake, I'll use a copy.